src/docs/resources/platform-adaptations.md

```diff
@@ -0,0 +1,218 @@
+          iOS push transition
+        </figcaption>
+      </figure>
+    </div>
+    <div class="col-sm text-center">
+      <figure class="figure">
+        <img src="../../images/platform-adaptations/navigation-ios-modal.gif" class="figure-img img-fluid rounded" alt="An animation of the bottom-up style present page transition on iOS" />
+        <figcaption class="figure-caption">
+          iOS present transition
+        </figcaption>
+      </figure>
+    </div>
+  </div>
+</div>
+
+### Platform-Specific Transition Details
+
+On **Android**, 2 page transition animation styles exist depending on your OS
+version:
+
+* Pre API 28 uses a bottom-up animation that [slides up and fades in](https://docs.flutter.io/flutter/material/FadeUpwardsPageTransitionsBuilder-class.html). 
+* On API 28 and later, the bottom-up animation [slides and clip-reveals up](https://docs.flutter.io/flutter/material/OpenUpwardsPageTransitionsBuilder-class.html).
+
+On **iOS**, Flutter's bundled [CupertinoNavigationBar](https://docs.flutter.io/flutter/cupertino/CupertinoNavigationBar-class.html)
+and [CupertinoSliverNavigationBar](https://docs.flutter.io/flutter/cupertino/CupertinoSliverNavigationBar-class.html)
+will automatically animate each of its subcomponent to its corresponding
+subcomponent on the next or previous page's CupertinoNavigationBar or
+CupertinoSliverNavigationBar when the push style transition is used.
+
+<div class="container">
+  <div class="row">
+    <div class="col-sm text-center">
+      <figure class="figure">
+        <img src="../../images/platform-adaptations/navigation-android.gif" class="figure-img img-fluid rounded" alt="An animation of the page transition on Android pre-Android P" />
+        <figcaption class="figure-caption">
+          Android Pre-P
+        </figcaption>
+      </figure>
+    </div>
+    <div class="col-sm">
+      <figure class="figure text-center">
+        <img src="../../images/platform-adaptations/navigation-android-p.gif" class="figure-img img-fluid rounded" alt="An animation of the page transition on Android on Android P" />
+        <figcaption class="figure-caption">
+          Android Post-P
+        </figcaption>
+      </figure>
+    </div>
+    <div class="col-sm">
+      <figure class="figure text-center">
+        <img src="../../images/platform-adaptations/navigation-ios-nav-bar.gif" class="figure-img img-fluid rounded" alt="An animation of the nav bar transitions during a page transition on iOS" />
+        <figcaption class="figure-caption">
+          iOS Nav Bar
+        </figcaption>
+      </figure>
+    </div>
+  </div>
+</div>
+
+### Back Navigation
+
+On **Android**, the OS back button, by default, is sent to Flutter and pops the top
+route of the [WidgetsApp](https://docs.flutter.io/flutter/widgets/WidgetsApp-class.html)'s
+Navigator.
+
+On **iOS**, an edge swipe gesture can be used to pop the top route.
+
+## Scrolling
+
+### Physics Simulation
```

Comment:
=> Physics simulation

Can you add a sentence here, before the L3 header? Because a) there should be one and b) anchors don't work right if there's no text.  (This is a global change. I've gone back and looked for it.)